test/NuGet.Clients.Tests/NuGet.CommandLine.Test/MockServer.cs

```diff
@@ -188,6 +188,11 @@ public static void SavePushedPackage(HttpListenerRequest r, string outputFileNam
             }
         }
 
+        public static bool PushedPackageExists(string outputFileName)
```

Comment:
Where is this method used? I didn't find any references to it by using Find All References in VS.